ivy/functional/backends/paddle/experimental/creation.py

```diff
@@ -30,7 +30,7 @@ def _kaiser_window(window_length, beta):
 
 def kaiser_window(
     window_length: int,
-    periodic: bool = True,
+    periodic: bool = False,
```

Comment:
please revert this as the function signature should be consistent across all backends
```suggestion
    periodic: bool = True,
```